src/components/SearchBar/SearchbarIcons.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import './searchbar.css';
+
+const SearchbarIcons = props => {
+    return(
+        <div className={'iconsContent'}>
```

Comment:
I would suggest adding tabs or some sort of styling to make this code more readable.